<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="use_simulation_gui" default="True"/>
  <arg name="cad_models_additional_search_path" default="$(find uwds3)/models/cad_models"/>
  <arg name="static_entities_config_filename" default="$(find uwds3)/config/simulation/adream_static_entities.yaml"/>
  <arg name="simulation_config_filename" default="$(find uwds3)/config/simulation/r2d2_config.yaml"/>
  <arg name="ouput_data_directory" default="$(find uwds3)/data/adream_snapshot/"/>

  <node name="data_generator" pkg="uwds3" type="data_generator_node.py" args="" output="screen" respawn="false">
    <rosparam subst_value="true">

      use_simulation_gui: $(arg use_simulation_gui)
      simulation_config_filename: $(arg simulation_config_filename)
      cad_models_additional_search_path: $(arg cad_models_additional_search_path)
      static_entities_config_filename: $(arg static_entities_config_filename)
      ouput_data_directory: $(arg ouput_data_directory)

    </rosparam>
  </node>
</launch>
